components/ChatVisualization/ChatVisualization.vue

```diff
@@ -107,51 +97,52 @@ export default {
       showDownloadPopup: false,
       ego: this.chat.messagesPerPerson[0].name,
       isLoading: false,
+      price: 0.99,
+      currency: "EUR",
```

Comment:
If we post on data is beautiful, maybe $ is better? As the main user group from Reddit are Americans, https://www.statista.com/statistics/325144/reddit-global-active-user-distribution/

Reply:
Nah, We need to switch currencies in EUR and then we need to take the rather low PayPal currency exchange rates.